frontend/src/utilities/useTableColumnSort.ts

```diff
@@ -49,7 +49,7 @@ const useTableColumnSort = <T>(
 
         const compute = () => {
           if (typeof columnField.sortable === 'function') {
-            return columnField.sortable(a, b, columnField.field);
+            return columnField.sortable(a, b, columnField.field, activeSortDirection);
```

Comment:
Why did you do this? This should be supported by [line 71](https://github.com/opendatahub-io/odh-dashboard/pull/1099/files#diff-43887b422d0129bb706fb8c6a70e86b36f84c8f3269876b8664cdfca6e750c97R71)

Reply:
It's to expose the direction for the callback, with that, I can get the workaround of keeping the creation field on the bottom. See the gif:

![sort](https://user-images.githubusercontent.com/16117276/234551267-bdbf5cd4-a3bf-486c-b160-fd07d22a2d18.gif)

@kywalker-rh if we don't want this behavior I can revert it back.

Reply:
Not a UX thing -- don't worry about it Kyle.

@lucferbux Yeah -- we'll need to find a way to keep it at the bottom. I think we may need to just create a "sticky footer" concept for tables: https://github.com/opendatahub-io/odh-dashboard/pull/1099#discussion_r1177760688